mne/io/cnt/cnt.py

```diff
@@ -0,0 +1,404 @@
+    read_blocks : bool
+        Whether to read data in blocks. This is for dealing with different
+        kinds of CNT data formats.
+    preload : bool or str (default False)
+        Preload data into memory for data manipulation and faster indexing.
+        If True, the data will be preloaded into memory (fast, requires
+        large amount of memory). If preload is a string, preload is the
+        file name of a memory-mapped file which is used to store the data
+        on the hard drive (slower, requires less memory).
+    verbose : bool, str, int, or None
+        If not None, override default verbose level (see mne.verbose).
+
+    Returns
+    -------
+    Instance of RawCNT.
+    """
+    return RawCNT(input_fname, eog=eog, ecg=ecg, emg=emg, misc=misc,
+                  read_blocks=read_blocks, preload=preload, verbose=verbose)
+
+
+def _get_cnt_info(input_fname, eog, ecg, emg, misc, read_blocks):
+    """Helper for reading the cnt header."""
+    data_offset = 900  # Size of the 'SETUP' header.
+    cnt_info = dict()
+    # Reading only the fields of interest. Structure of the whole header at
+    # http://paulbourke.net/dataformats/eeg/
+    with open(input_fname, 'rb', buffering=0) as fid:
+        fid.seek(21)
+        patient_id = _read_str(fid, 20)
+        patient_id = int(patient_id) if patient_id.isdigit() else 0
+        fid.seek(121)
+        patient_name = _read_str(fid, 20)
+        last_name = patient_name[0]
+        first_name = patient_name[-1]
+        fid.seek(2, 1)
+        sex = _read_str(fid, 1)
+        if sex == 'M':
+            sex = FIFF.FIFFV_SUBJ_SEX_MALE
+        elif sex == 'F':
+            sex = FIFF.FIFFV_SUBJ_SEX_FEMALE
+        else:  # can be 'U'
+            sex = FIFF.FIFFV_SUBJ_SEX_UNKNOWN
+        hand = _read_str(fid, 1)
+        if hand == 'R':
+            hand = FIFF.FIFFV_SUBJ_HAND_RIGHT
+        elif hand == 'L':
+            hand = FIFF.FIFFV_SUBJ_HAND_LEFT
+        else:  # can be 'M' for mixed or 'U'
+            hand = FIFF.FIFFV_SUBJ_HAND_UNKNOWN
+        fid.seek(205)
+        session_label = _read_str(fid, 20)
+        session_date = _read_str(fid, 10)
+        time = _read_str(fid, 12)
+        date = session_date.split('/')
+        if len(date) != 3:
+            warn('Could not parse meas date from the header. Setting to 0...')
+            meas_date = 0
+        else:
+            if date[2].startswith('9'):
+                date[2] = '19' + date[2]
+            elif len(date[2]) == 2:
+                date[2] = '20' + date[2]
+            time = time.split(':')
+            # Assuming mm/dd/yy
+            date = datetime.datetime(int(date[2]), int(date[0]), int(date[1]),
+                                     int(time[0]), int(time[1]), int(time[2]))
+            meas_date = calendar.timegm(date.utctimetuple())
+            if meas_date < 0:
+                warn('Could not parse meas date from the header. Setting to '
+                     '0...')
```

Comment:
hmm ... I was suggesting not to set it. But if that's not possible, okay with current solution.